src/Tizen.NUI.Components/Controls/AlertDialog.cs

```diff
@@ -190,6 +190,12 @@ private string InternalTitle
                 {
                     textLabel.Text = title;
                 }
+
+                if (Accessibility.Accessibility.IsEnabled && IsHighlighted)
+                {
+                    AccessibilityName = title;
```

Comment:
Updating `AccessibilityName` should be moved out of this `if` statement. Otherwise, its value would be outdated if Screen Reader is turned on dynamically after this code is executed (or if the name is changed while the control is not highlighted), resulting in wrong TTS feedback.